BricksAndHearts.UnitTests/ControllerTests/Property/PropertyControllerTests.cs

```diff
@@ -310,28 +310,29 @@ public void ViewProperty_WithExistingProperty_ReturnViewPropertyView()
         result!.ViewData.Model.Should().BeOfType<PropertyViewModel>();
     }
     
+    /*// This is not working

```

Comment:
Something like this seems to work OK:

```cs
        var urlHelper = A.Fake<IUrlHelper>();
        // ReSharper disable once Mvc.ActionNotResolved
        A.CallTo(() => urlHelper.Action("GetImage", new { propertyId = 1, fileName = "TestFileName" })!).Returns("/fake/url");
        UnderTest.Url = urlHelper;
```

That said I'd like to get this merged, so feel free to just delete the test for now if it's annoying.

Reply:
No I tried but it's still not working... I've commented the code out and perhaps raise a ticket for that?

Reply:
Yeah sure :+1: